download-templates.sh

```diff
@@ -9,6 +9,13 @@ manifest_url="https://github.com/kubevirt/kubevirt/releases/download"
 
 if [ -z "${version}" ]; then
    version=$(git ls-remote --tags https://github.com/kubevirt/kubevirt |  sed 's/.*refs\/tags\///g' | grep -v '\^{}')
+   major=`echo $version | cut -d. -f2`
+fi
+
+if [ "$major" -lt "7" ] ; then 
+  vm_samples_dir="cluster"
+else
+  vm_samples_dir="cluster/examples"
```

Comment:
To help remove these if blocks later we should have an explicit comment 

```
#remove after we stop supporting kubevirt < 0.7.x.
```

Reply:
@karmab can you add this comment.  Then will merge.